sippy-ng/src/jobs/JobAnalysis.js

```diff
@@ -89,18 +89,19 @@ export function JobAnalysis(props) {
       fetch(
         `${process.env.REACT_APP_API_URL}/api/jobs/analysis?${queryParams}`
       ),
-      fetch(`${process.env.REACT_APP_API_URL}/api/jobs/bugs?${queryParams}`),
     ])
-      .then(([analysis, bugs]) => {
+      .then(([analysis]) => {
         if (analysis.status !== 200) {
           throw new Error('server returned ' + analysis.status)
         }
 
-        return Promise.all([analysis.json(), bugs.json()])
+        return Promise.all([analysis.json()])
       })
-      .then(([analysis, bugs]) => {
+      .then(([analysis]) => {
         setAnalysis(analysis)
-        setBugs(bugs)
+        setBugsURL(
+          `${process.env.REACT_APP_API_URL}/api/jobs/bugs?${queryParams}`
+        )
```

Comment:
Didn't want to copy the queryParams logic to BugTable.  Could have set that but figured passing in URL was more generic.